docs/src/making-changes.md

```diff
@@ -14,75 +14,88 @@ shell> git config --global user.email "EMAIL"
 
 ### Documentation changes
 
-If you want to improve the online documentation of a package, the easiest approach (at least for small changes) is to use GitHub's online editing functionality. First, navigate to the repository's GitHub "home page," find the file (e.g., `README.md`) within the repository's folder structure, and click on it. You'll see the contents displayed, along with a small "pencil" icon in the upper right hand corner. Clicking that icon opens the file in edit mode. Make your changes, write a brief summary describing the changes you want to make (this is your *commit message*), and then hit "Propose file change." Your changes will be submitted for consideration by the package owner(s) and collaborators.
+If you want to improve the online documentation of a package, the easiest approach for small changes is to use GitHub's online editing functionality. First, navigate to the repository's GitHub "home page," find the file (e.g., `README.md`) within the repository's folder structure, and click on it. You'll see the contents displayed, along with a small "pencil" icon in the upper right hand corner. Clicking that icon opens the file in edit mode. Make your changes, write a brief summary describing the changes you want to make (this is your *commit message*), and then hit "Propose file change." Your changes will be submitted for consideration by the package owner(s) and collaborators.
 
 For larger documentation changes--and especially ones that you expect to have to update in response to feedback--you might find it easier to use the procedure for code changes described below.
 
 ### Code changes
 
 #### Executive summary
 
-Here we assume you've already set up git on your local machine and have a GitHub account (see above). Let's imagine you're fixing a bug in the `Images` package:
+Here we assume you've already set up git on your local machine and have a GitHub account (see above). Let's imagine you're fixing a bug in the `Example` package. First, fork the project since you don't have permission to push directly to the repository: navigate to the [`Example.jl`](https://github.com/JuliaLang/Example.jl) GitHub page and fork the project using the botton on the top right hand side of the page. Then fix the code and make a pull request (or PR in short).
 
 ```jl
-Pkg.checkout("Images")           # check out the master branch
-<here, make sure your bug is still a bug and hasn't been fixed already>
-cd(Pkg.dir("Images"))
-;git checkout -b myfixes         # create a branch for your changes
-<edit code>                      # be sure to add a test for your bug
-Pkg.test("Images")               # make sure everything works now
-;git commit -a -m "Fix foo by calling bar"   # write a descriptive message
-using PkgDev
-PkgDev.submit("Images")
+(v1.1) pkg> add Example#master           # check out the master branch
+(v1.1) pkg> develop --local Example      # make a local copy
+# change path to the directory
+shell> cd(?)
```

Comment:
The Pkg documentation mentions the variable `JULIA_PKG_DEVDIR` which is not defined if one types `ENV["JULIA_PKG_DEVDIR"]`. Is there an easy way to access it or would it make sense to simply define it?